include/deal.II/lac/linear_operator_tools.h

```diff
@@ -0,0 +1,30 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__linear_operator_tools_h
+#define dealii__linear_operator_tools_h
+
+// many usage cases lead to a combination of LinearOperator and
+// PackagedOperation. To ease the pain of reading compilation errors, just include
+// all headers we ever need to use LO and friends in one place:
+#include <deal.II/lac/linear_operator.h>
+#include <deal.II/lac/block_linear_operator.h>
+#include <deal.II/lac/packaged_operation.h>
+#include <deal.II/lac/constrained_linear_operator.h>
+
+// keep Schur complement for completeness
+#include <deal.II/lac/schur_complement.h>
```

Comment:
Oh, wow. I totally forgot that ```schur_complement.h``` is implemented in terms of ```LinearOperator```. Can you just strip the comment and group it with the ```constrained_linear_operator.h``` include?